packages/adapters/ethers5/src/client.ts

```diff
@@ -828,8 +827,8 @@ export class Ethers5Adapter {
     }
 
     this.appKit?.setLoading(true)
-    const chainId = NetworkUtil.caipNetworkIdToNumber(this.appKit?.getCaipNetwork()?.id)
-    this.appKit?.setCaipAddress(`eip155:${chainId}:${address}`, this.chainNamespace)
+    const chainId = this.appKit?.getCaipNetwork()?.id
+    this.appKit?.setCaipAddress(`${this.chainNamespace}:${chainId}:${address}`, this.chainNamespace)
```

Comment:
same as before

Reply:
Same, we should avoid hard coded strings